CHANGELOG

```diff
@@ -1,5 +1,17 @@
+v0.4.12 (3/22/16)
```

Comment:
I thought we were talking about making this 0.5.0?
We bumped up to 0.4.0 when we added the visual diff and a11y testing features.